tools/src/main/java/org/apache/kafka/trogdor/rest/JsonRestServer.java

```diff
@@ -197,6 +217,26 @@ public void stop() {
         }
     }
 
+    public static <T> HttpResponse<T> httpRequest(String url, String method,
+            Object requestBodyData, TypeReference<T> responseFormat, int maxTries)
+            throws IOException, InterruptedException {
+        IOException exc = null;
+        for (int tries = 0; tries < maxTries; tries++) {
+            if (tries > 1) {
+                Thread.sleep(10);
+            } else if (tries > 0) {
+                Thread.sleep(2);
+            }
```

Comment:
Hmm-- I might be misunderstanding this comment.  Wouldn't that be more verbose than what is here?

It would have to be like

            if (tries > 0) {
              if (tries > 1) {
                  sleepTime = 10;
              } else {
                 sleepTime = 2;
              }
              Thread.sleep(sleepTime);
           }

Reply:
I was thinking:
```
if (tries > 0)
    Thread.sleep(tries > 1 ? 10 : 2);
```